packages/core/integration-tests/test/server.js

```diff
@@ -107,7 +107,11 @@ describe('server', function() {
 
   it('should serve a 500 if the bundler errored', async function() {
     let port = await getPort();
-    let b = bundler(path.join(__dirname, '/integration/commonjs/index.js'), {
+    let inputDir = path.join(__dirname, '/input/server-500');
+    await fs.ncp(path.join(__dirname, '/integration/commonjs'), inputDir);
```

Comment:
Looks like a promisified ncp already exists in test utils? Do you think we'll need it outside of tests? Should probably remove from test utils if so, but not a blocker or anything.